paimon-web-server/src/test/java/org/apache/paimon/web/server/controller/CatalogControllerTest.java

```diff
@@ -39,7 +39,7 @@
 @SpringBootTest
 @AutoConfigureMockMvc
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
-public class CatalogControllerTest extends ControllerTestBase {
+public class CatalogControllerTest extends MetadataTestBase {
```

Comment:
Why does `CatalogControllerTest` extend `MetadataTestBase`? IMO, you could improve the `ControllerTestBase`.